code/AdminController.php

```diff
@@ -325,17 +326,13 @@ protected function jsonError(int $errorCode, string $errorMessage = ''): void
         if ($errorMessage) {
             $error['value'] = $errorMessage;
         } else {
-            $messageDefault = match ($errorCode) {
-                400 => 'Sorry, it seems there was something wrong with the request.',
-                401 => 'Sorry, it seems you are not authorised to access this section or object.',
-                403 => 'Sorry, it seems the action you were trying to perform is forbidden.',
-                404 => 'Sorry, it seems you were trying to access a section or object that doesn\'t exist.',
-                500 => 'Sorry, it seems there was an internal server error.',
-                503 => 'Sorry, it seems the service is temporarily unavailable.',
-                default => 'Error',
-            };
-            /** @phpstan-ignore translation.key (we need the key to be dynamic here) */
-            $error['value'] = _t(__CLASS__ . ".ErrorMessage{$errorCode}", $messageDefault);
+            $messageDefault = $this->getErrorCodeDefaultMessages()[$errorCode] ?? null;
+            if ($messageDefault) {
+                /** @phpstan-ignore translation.key (we need the key to be dynamic here) */
+                $error['value'] = _t(__CLASS__ . ".ErrorMessage{$errorCode}", $messageDefault);
+            } else {
+                $error['value'] = _t(__CLASS__ . ".ErrorMessage", 'Sorry, it seems there was a {errorcode} error.', $errorCode);
```

Comment:
This default was missing when the code was ported over from `LeftAndMain`. The localisation key was already there, just forgot to keep the message itself lol.